File: src/bubble.rs
pub fn sort(list: &mut [u8]) {
    let mut swapped = true;
    while swapped {
        swapped = false;
        for i in 1..list.len() {
            if list[i] < list[i - 1] {
                list.swap(i, i - 1);
                swapped = true;
            }
        }
    }
}

File: python/sorting/bubble.py
def sort(list):
    swapped = True
    while swapped:
        swapped = False
        for i in range(1,len(list)):
            if list[i] < list[i-1]:
                list[i], list[i-1] = list[i-1], list[i]
                swapped = True
    return list

